vnext/Microsoft.ReactNative/TurboModulesProvider.cpp

```diff
@@ -384,10 +436,6 @@ std::vector<std::string> TurboModulesProvider::getEagerInitModuleNames() noexcep
   if (it != m_moduleProviders.end()) {
     eagerModules.push_back("UIManager");
   }
-  it = m_moduleProviders.find("FabricUIManagerBinding");
```

Comment:
Why is this removed here and not in the backported changes to 73/72?

Reply:
This looks like a merge conflict.  This code shouldn't be removed.

Reply:
It is a bad merge. Thank you for catching it!

Reply:
The lines are added back